spec/services/reports/mis/non_passported_applications_report_spec.rb

```diff
@@ -5,9 +5,18 @@ module MIS
     RSpec.describe NonPassportedApplicationsReport do
       before do
         create_early_non_passported_application
-        Timecop.freeze(8.minutes.ago) { create_non_passported_application }
-        Timecop.freeze(6.minutes.ago) { create_non_passported_application_use_ccms }
-        Timecop.freeze(4.minutes.ago) { create_passported_application }
+        travel_back
+        travel(8.minutes) { create_non_passported_application }
```

Comment:
Does this change the sequence? 
Previously we had 8.minutes.ago, now we have 9 minutes in the future.
Is this why the expected values changed at line 30?

Reply:
Yes that's it XD. Great spot. I meant to use travel_to and 8.minutes ago to go to the past.